impeller/display_list/aiks_dl_path_unittests.cc

```diff
@@ -432,8 +433,8 @@ TEST_P(AiksTest, ArcWithZeroSweepAndBlur) {
   DlPaint paint;
   paint.setColor(DlColor::kRed());
 
-  std::vector<DlColor> colors = {DlColor{1.0, 0.0, 0.0, 1.0},
-                                 DlColor{0.0, 0.0, 0.0, 1.0}};
+  std::vector<DlColor> colors = {DlColor::RGBA(1.0, 0.0, 0.0, 1.0),
```

Comment:
This looked like a bug and I was wondering why it still worked when I switched the factory, but it looks like the reason is that this isn't a golden test - it's a "can we call builder.Build() without crashing" test in which case the colors don't matter...